backend/cn/testGeneration.ml

```diff
@@ -1,3 +1,9 @@
+(** Test Generation
+    Handles all parts of test generation from
+    gathering constraints to building
+    generators and finally exporting C++ code.
```

Comment:
This comment doesn't really explain what "test generation" means, or what the high-level purpose of it being in CN and using the specifications/types.